application-services/custom/influxdb-export/Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: build clean
+
+GO=CGO_ENABLED=1 go
+
+# VERSION file is not needed for local development, In the CI/CD pipeline, a temporary VERSION file is written
```

Comment:
Should remove this and just set APPVERSION=1.3.0 since example don't have CI/CD?